src/components/radio-button/radio-button-test.stories.tsx

```diff
@@ -138,52 +183,52 @@ export const WithTooltipPositionOnRadioGroup: StoryFn<
 WithTooltipPositionOnRadioGroup.storyName =
   "with tooltip position on RadioGroup";
 
-const radioContainerWidth = 400;
-
-export const RadioButtonComponent = (props: Partial<RadioButtonProps>) => {
-  const [isChecked, setIsChecked] = React.useState(false);
+export const WithNewValidation = (props: Partial<RadioButtonProps>) => {
```

Comment:
**suggestion(non-blocking)**: what do you think about removing the `WithNewValidation` and `WithNewValidationGroup` test stories, in favour of the consumer-facing stories `NewValidationDefault` and `NewValidationDefaultGroup`? It appears the latter two will be captured by a Chromatic snapshot, whereas the former two won't (disabled on line 23).